tests/test_flags

```diff
@@ -1,2 +1,5 @@
 # One line per flag
-# For example --someflag=value
\ No newline at end of file
+# For example --someflag=value
```

Comment:
is this actually operational?  will this run if i run `python -m unittest` etc ?

Reply:
It only gets pulled in if you use `./test.sh`